prow/plugins/bugzilla/bugzilla.go

```diff
@@ -819,6 +827,65 @@ func handleMerge(e event, gc githubClient, bc bugzilla.Client, options plugins.B
 	return comment(fmt.Sprintf("%s %s\n%s", mergedMessage("Some"), unmergedMessage, outcomeMessage("")))
 }
 
+func handleCherrypick(e event, gc githubClient, bc bugzilla.Client, options plugins.BugzillaBranchOptions, log *logrus.Entry) error {
+	comment := e.comment(gc)
+	// get the info for the PR being cherrypicked from
+	pr, err := gc.GetPullRequest(e.org, e.repo, e.cherrypickFromPRNum)
+	if err != nil {
+		log.WithError(err).Warn("Unexpected error getting title of pull request being cherrypicked from.")
+		return comment(formatError(fmt.Sprintf("checking the state of a cherrypicked pull request at https://github.com/%s/%s/pull/%d", e.org, e.repo, e.cherrypickFromPRNum), bc.Endpoint(), e.bugId, err))
+	}
+	// Attempt to identify bug from PR title
+	bugID, bugMissing, err := bugIDFromTitle(pr.Title)
+	if bugMissing {
+		log.Debugf("Parent PR %d doesn't have associated bug; not creating cherrypicked bug", pr.Number)
+		// if there is no bugzilla bug, we should simply ignore this PR
+		return nil
+	} else if err != nil {
+		// should be impossible based on the regex
+		log.WithError(err).Debugf("Failed to get bug ID from PR title \"%s\"", pr.Title)
+		return comment(formatError(fmt.Sprintf("Failed to get bug ID from PR title \"%s\"", pr.Title), bc.Endpoint(), 0, err))
```

Comment:
maybe we want to add that prefix to explain what we were doing to all of these comments actually